conf/application.conf

```diff
@@ -107,15 +95,13 @@ session {
 
 mongodb {
   uri                 = "mongodb://localhost:27017/"${appName}
-  timeToLiveInSeconds = 900
```

Comment:
Why are we removing this?

Reply:
don't think this is being used. the only frontend mongo repository is counter and has no ttl from what can tell. Unless this is being used in some other way perhaps?

Reply:
As discussed adding back in due to feeling uneasy when it comes to mongo and ticket is to just fix the warnings. Small clean up for now but leaving it in.